backend/service/authn/crypto.go

```diff
@@ -0,0 +1,64 @@
+package authn
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"crypto/sha256"
+	"fmt"
+)
+
+// AES-256 encryption.
+type cryptographer struct {
+	gcm cipher.AEAD
+}
+
+func newCryptographer(passphrase string) (*cryptographer, error) {
+	if len(passphrase) == 0 {
```

Comment:
should we use something here to enforce password complexity? Maybe [zxcvbn](https://github.com/nbutton23/zxcvbn-go)?

Reply:
we could but maybe there's some confusion? this is not a user password. it is an encryption passphrase configured on the server. seems a little handholdy to enforce some arbitrary strength constraints on the clutch administrator. however if we want to ensure a stronger key we could maybe try to do so with protoc-gen-validate annotations rather than introducing a separate mechanism.

Reply:
sounds good to me. I follow the source of the passphrase but since this will be potentially user generated I was wondering if it would make sense to ensure that they don't dump in something simple such as a single digit or character